crates/database/src/quotes.rs

```diff
@@ -141,6 +141,7 @@ mod tests {
     async fn postgres_save_and_get_quote_by_id() {
         let mut db = PgConnection::connect("postgresql://").await.unwrap();
         let mut db = db.begin().await.unwrap();
+        crate::clear_DANGER_(&mut db).await.unwrap();
```

Comment:
nit: Shouldn't this function be called `clear_db` or similar?

Reply:
It is in the database crate so I didn't feel we needed an extra pre/postfix.